src/libs/web3-data-provider/Web3Provider.tsx

```diff
@@ -148,14 +148,16 @@ export const Web3ContextProvider: React.FC<{ children: ReactElement }> = ({ chil
     const { ethereum } = window;
 
     if (!ethereum?.providers) {
-      return false;
+      return true;
```

Comment:
if there are no `providers` that means there is only one, so we should connect to it